kickoff/jsonexport.py

```diff
@@ -277,3 +280,54 @@ def json_exports_txt():
     response = make_response(render_template("json_exports.txt", jsonFiles=jsonFiles))
     response.mimetype = "text/plain"
     return response
+
+
+def get_releases_for_json(product):
```

Comment:
Should this be consistent with the rest of the naming and be `getReleasesForJson`?